services/src/main/scala/cromwell/services/metadata/WorkflowQueryKey.scala

```diff
@@ -50,23 +61,30 @@ object WorkflowQueryKey {
     }
   }
 
-  case object LabelKeyValue extends SeqWorkflowQueryKey[Label] {
-    override val name = "Label"
-
+  sealed trait LabelLikeKeyValue extends SeqWorkflowQueryKey[Label] {
     override def validate(grouped: Map[String, Seq[(String, String)]]): ErrorOr[List[Label]] = {
       val values = valuesFromMap(grouped).toList
 
       def validateLabelRegex(labelKeyValue: String): ErrorOr[Label] = {
         labelKeyValue.split("\\:", 2) match {
           case Array(k, v) => Label.validateLabel(k, v)
-          case other @ _ => s"$labelKeyValue".invalidNel
+          case _ => labelKeyValue.invalidNel
         }
       }
       val nels: List[ErrorOr[Label]] = values map validateLabelRegex
       sequenceListOfValidatedNels("Label values do not match allowed pattern label-key:label-value", nels)
     }
   }
 
+
+  case object LabelAndKeyValue extends LabelLikeKeyValue {
+    override val name = "Label"
```

Comment:
`Labeland` for symmetry or would that be breaking?

Reply:
`Labeland` instead of `Label` would be breaking unfortunately. In theory we could support both... or have a new versioned endpoint... but 🏈